test/test_advisory.py

```diff
@@ -1,5 +1,6 @@
 from json import loads
 
+import pytest
```

Comment:
can we just import what we really need? like mark?

Reply:
Also fixed